test/e2e/play_kube_test.go

```diff
@@ -1137,6 +1137,30 @@ var _ = Describe("Podman play kube", func() {
 		Expect(infraContainerImage).To(Equal(config.DefaultInfraImage))
 	})
 
+	It("podman play kube should use default infra_image", func() {
```

Comment:
Is this actually a test of the default infra image? It looks like it's testing no-hosts only

Reply:
No cut and paste, fixed.